packages/suite/src/middlewares/suite/__tests__/buttonRequestMiddleware.test.ts

```diff
@@ -63,6 +63,8 @@ describe('buttonRequest middleware', () => {
         store.getActions().pop();
         expect(store.getActions()).toMatchObject([
             { type: connectInitThunk.pending.type, payload: undefined },
+            { type: getBinFilesBaseUrlThunk.pending.type },
+            { type: getBinFilesBaseUrlThunk.fulfilled.type },
```

Comment:
The order isn't random, so I believe this will not be flaky.
First connectInit gets initiated, then getBinFiles but gets resolved first (`createThunk` is always async, but in this case there is nothing async going on and it gets resolved immediately)